astropy/io/ascii/qdp.py

```
@@ -0,0 +1,559 @@
+
+    names : list of strings
+        Name of data columns (defaults to ['col1', 'col2', ...]), _not_
+        including error columns.
+
+    Returns
+    -------
+    tables : list of `Table` objects
+        List containing all the tables present inside the QDP file
+    """
+    if table_id is None:
+        warnings.warn("table_id not specified. Reading the first available "
```

Ruff (0.11.9)

No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)